spring-data-multi-storage-transaction-sample/README.md

```diff
@@ -0,0 +1,286 @@
+    }
+  },
+  "order.statements": {
+    "transaction": true,
+    "partition-key": [
+      "order_id"
+    ],
+    "clustering-key": [
+      "item_id"
+    ],
+    "columns": {
+      "order_id": "TEXT",
+      "item_id": "INT",
+      "count": "INT"
+    }
+  },
+  "order.items": {
+    "transaction": true,
+    "partition-key": [
+      "item_id"
+    ],
+    "columns": {
+      "item_id": "INT",
+      "name": "TEXT",
+      "price": "INT"
+    }
+  }
+}
+```
+
+All the tables are created in the `customer` and `order` namespaces.
+
+- `customer.customers`: a table that manages customers' information
+  - `credit_limit`: the maximum amount of money a lender will allow each customer to spend when using a credit card
+  - `credit_total`: the amount of money that each customer has already spent by using the credit card
+- `order.orders`: a table that manages order information
+- `order.statements`: a table that manages order statement information
+- `order.items`: a table that manages information of items to be ordered
+
+The Entity Relationship Diagram for the schema is as follows:
+
+![ERD](images/ERD.png)
+
+### Transactions
+
+The following five transactions are implemented in this sample application:
+
+1. Getting customer information
+2. Placing an order by credit card (checks if the cost of the order is below the credit limit, then records order history and updates the `credit_total` if the check passes)
+3. Getting order information by order ID
+4. Getting order information by customer ID
+5. Repayment (reduces the amount in the `credit_total`)
+
+### Configuration
+
+Configurations for the sample Spring Boot application are as follows:
+
+```application.properties
+spring.datasource.driver-class-name=com.scalar.db.sql.jdbc.SqlJdbcDriver
+spring.datasource.url=jdbc:scalardb:\
+?scalar.db.storage=multi-storage\
+&scalar.db.multi_storage.storages=cassandra,mysql\
+&scalar.db.multi_storage.storages.cassandra.storage=cassandra\
+&scalar.db.multi_storage.storages.cassandra.contact_points=localhost\
+&scalar.db.multi_storage.storages.cassandra.username=cassandra\
+&scalar.db.multi_storage.storages.cassandra.password=cassandra\
+&scalar.db.multi_storage.storages.mysql.storage=jdbc\
+&scalar.db.multi_storage.storages.mysql.contact_points=jdbc:mysql://localhost:3306/\
+&scalar.db.multi_storage.storages.mysql.username=root\
+&scalar.db.multi_storage.storages.mysql.password=mysql\
+&scalar.db.multi_storage.namespace_mapping=customer:mysql,order:cassandra,coordinator:cassandra\
+&scalar.db.multi_storage.default_storage=cassandra
+```
+
+- `scalar.db.storage`: `multi-storage` to use Multi-storage Transaction in ScalarDB
+- `scalar.db.multi_storage.storages`: Definition of storage names
+- `scalar.db.multi_storage.storages.cassandra.*`: Configurations of `cassandra` storage that is one of the storages defined in `scalar.db.multi_storage.storages`. You can configure all the properties of `scalar.db.*` for the `cassandra` storage here
```

Comment:
This was a bit tricky to reword. The word "storage" is similar to the word "data" in that, even if there are multiple storage units, the word doesn't become plural.

If the wording in my suggestion is incorrect, let's find a way to reword it!

```suggestion
- `scalar.db.multi_storage.storages.cassandra.*`: These configurations are for the `cassandra` storage, which is one of the storage types defined in `scalar.db.multi_storage.storages`. You can configure all the `scalar.db.*` properties for the `cassandra` storage here.
```

Reply:
Looks perfect to me. Thanks!!